cogs/staff.py

```diff
@@ -364,7 +364,9 @@ async def searchModLogs(self, ctx: commands.Context, *, search_term: str = comma
                     # Enum list, and format message
                     [f"{i+1}. {msg['url']}: `{msg['context']}`" for i, msg in enumerate(messageLinksList[::-1])]
                 )
-                await ctx.send(f"Moderation Logs related to search term: {search_term}\n{messageLinks}"[:2000]) # Discord message limit
+                embed = discord.Embed(title=f"Moderation logs for: {search_term}", description=f"{messageLinks}"[:4000], color=discord.Color.red())
```

Comment:
Neutral color (gold).
Backticks around user input.

```suggestion
                embed = discord.Embed(title=f"Moderation logs for: `{search_term}`", description=f"{messageLinks}"[:4000], color=discord.Color.gold())
```